### PR TITLE
Replaced screenshots for triggers

### PR DESCRIPTION
**Pull request by ralf401**

### CODE REVIEW

File: images/manage/organizations/general-view-of-a-busy-organization.png
Comment (dvuckovic) [inline images/manage/organizations/general-view-of-a-busy-organization.png:1]: The new screenshot seems a tad bit too large: let's recreate it in 1280x720 max.

File: images/manage/trigger/move-tickets-of-specific-customer-to-specific-group.png
Comment (dvuckovic) [inline images/manage/trigger/move-tickets-of-specific-customer-to-specific-group.png:1]: Same here: the dialog seems a bit too large.

File: images/manage/trigger/trigger-based-auto-replies-on-follow-ups.png
Comment (dvuckovic) [inline images/manage/trigger/trigger-based-auto-replies-on-follow-ups.png:1]: Size again.

File: images/manage/trigger/trigger-based-priotize-change.png
Comment (dvuckovic) [inline images/manage/trigger/trigger-based-priotize-change.png:1]: Size + typo in the filename.

**Review by dvuckovic — CHANGES_REQUESTED** [review]

File: images/manage/organizations/general-view-of-a-busy-organization.png
Comment (ralf401) [reply images/manage/organizations/general-view-of-a-busy-organization.png:1]: Oh it looks like my already smaller screenshot was overwritten somehow :-O

I just wanted to show the same information as before, but I think here it is no problem to limit it to 1280x720. Then the list of open / closed tickets will only show the title.

I re-upload it.

File: images/manage/trigger/move-tickets-of-specific-customer-to-specific-group.png
Comment (MrGeneration) [reply images/manage/trigger/move-tickets-of-specific-customer-to-specific-group.png:1]: Just my two cents:
There are several places in the documentations where you'll find screenshots that are drastically larger than the readme's limit at least for one side of the image. For the exact reason Ralf mentioned. When it was necessary information are shown, I usually used higher resolutions to overcome this or started to cut away uneeded UI parts like the tab bar (especially for triggers)

Comment (dvuckovic) [reply images/manage/trigger/move-tickets-of-specific-customer-to-specific-group.png:1]: I totally agree that in some cases the height of the screenshot has to be longer, however I would still try to create it against the width of 1280px. If we did our job correctly, all important elements should be contained in this width, and then the length only can be longer so everything of note is contained within.

As @MrGeneration mentioned, it's perfectly fine to screenshot just the relevant node (in this case a dialog), including the "chrome" of the app around it does not bear any purpose.

Comment (ralf401) [reply images/manage/trigger/move-tickets-of-specific-customer-to-specific-group.png:1]: I reduced it already, just showing the modal window and width is <1280px. So I will leave it as uploaded. I will double check the other ones, why they are different in width (not 1175px) and re-upload if necessary.

File: images/manage/trigger/trigger-based-auto-replies-on-follow-ups.png
Comment (ralf401) [reply images/manage/trigger/trigger-based-auto-replies-on-follow-ups.png:1]: Recreated screenshot. You are right, the modal was created without the limitation on 1280 in width.

**Review by dvuckovic — APPROVED** [review]

**Pull request merged**